debian/control

```diff
@@ -63,3 +66,12 @@ Description: JPEG XL Image Coding System - "JXL" (shared libraries)
  several features that help transition from the legacy JPEG format.
  .
  This package installs shared libraries.
+
+Package: libjxl-gimp
+Architecture: any
+Multi-Arch: same
+Section: universe/graphics
+Depends: ${shlibs:Depends}, ${misc:Depends}
```

Comment:
Probably should depend on the libjxl package and on gimp.

Reply:
I think debian does some magic and looks at the shared library dependencies of your binaries, that's what the shlibs:Depends does. For other cases (like in the .a to .a dependency in libjxl-dev to libhwy-dev) you have to spell it out; but otherwise... magic 🤯️

Reply:
I think the plugin may be linked statically.  On Windows, it works without any of the other libjxl dlls.

Reply:
no no, I checked - the dependency is added in the debian package automagically :)